minecraft_servers/pommesmc/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "server_name": "pommesmc",
+  "nice_name": "PommesMC",
+  "direct_ip": "pommesmc.tk",
+  "server_wildcards": [
+    "%.pommesmc.tk"
+  ],
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "twitter": "pommesdev",
+    "discord": "https://discord.gg/jraSytjU43"
+  },
+  "gamemodes": {
+    "citybuild": {
+      "name": "CityBuild",
+      "command": "/spawn",
+      "color": "#ff00f9"
+    }
+  },
+  "discord": {
+    "server_id": 847516968478441522,
+    "rename_to_minecraft_name": true

```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.